src/main/java/org/openstreetmap/atlas/generator/AtlasGeneratorHelper.java

```diff
@@ -427,28 +599,71 @@ public String getName()
             }
             catch (final Throwable e) // NOSONAR
             {
-                throw new CoreException(ERROR_MESSAGE,
-                        AtlasGeneratorJobGroup.WAY_SECTIONED_PBF.getDescription(), countryShardName,
-                        e);
+                logger.warn("Sectioning had error, probably was null atlas",
```

Comment:
Error, then logger.error? Or that happens too often?

Reply:
good catch!